lib/index.js

```diff
@@ -19,6 +19,20 @@ var render = require('posthtml-render')
  */
 function PostHTML (plugins) {
   this.plugins = plugins || []
+
+  /**
+   * @example
```

Comment:
`post-load-*` or any other plugin are unrelated to posthtml core docs and should be documented/mentioned elsewhere imho